server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -239,6 +241,10 @@ protected void closeInternal() {
             daemonThreadFactory(nodeName, DANGLING_INDICES_UPDATE_THREAD_NAME)) : null;
     }
 
+    public final void setSchemas(Schemas schemas) {
```

Comment:
We might be able to revisit this after https://github.com/crate/crate/pull/15988